tests/components/lifx/conftest.py

```diff
@@ -55,3 +55,15 @@ def device_reg_fixture(hass):
 def entity_reg_fixture(hass):
     """Return an empty, loaded, registry."""
     return mock_registry(hass)
+
+
+@pytest.fixture(autouse=True)
+async def mock_lifx_sleep():
+    """Mock out lifx sleeps."""
+    asyncio_sleep = asyncio.sleep
+
+    async def sleep(duration, loop=None):
+        await asyncio_sleep(0)
+
+    with patch("homeassistant.components.lifx.asyncio.sleep", new=sleep):
```

Comment:
I think you could patch it without replacing it

Reply:
I looked to see what other integrations' tests did with asyncio.sleep and this was the first example (in `atag`) and it worked, so I figured it was fine. If there's a better example of how to handle asyncio.sleep, let me know.

Reply:
I think you want to patch the the sleep in `homeassistant.components.lifx.light`


You could add this to `test_light.py`

```
@pytest.fixture(autouse=True)
def mock_lifx_light_sleep():
    """Mock out lifx light sleeps."""
    with patch("homeassistant.components.lifx.light.asyncio.sleep"):
        yield
```